sycl/source/detail/device_impl.cpp

```diff
@@ -222,6 +220,51 @@ pi_native_handle device_impl::getNative() const {
   return Handle;
 }
 
+bool device_impl::has(aspect Aspect) const {
+  switch (Aspect) {
+  case aspect::host:
+    return is_host();
+    break;
```

Comment:
I guess this `break` and others are redundant.

Reply:
Thanks!  Fixed this.